### PR TITLE
Final Pom Alignment for CI/CD

### PR DESCRIPTION
**Pull request by salaboy**
This PR contains changes to implement #1994 first iteration

### CODE REVIEW

**Review by erdemedeiros — APPROVED** [review]
It seems ok for me.

File: Jenkinsfile [hunk 1]
```diff
@@ -49,41 +42,24 @@ pipeline {
             // so we can retrieve the version in later steps
             sh "echo \$(jx-release-version) > VERSION"
             sh "mvn versions:set -DnewVersion=\$(cat VERSION)"
-            sh "mvn versions:set-property -Dproperty=activiti.version -DnewVersion=\$(cat VERSION)"
+
+            sh "git add --all"
+            sh "git commit -m 'release \$(cat VERSION)' --allow-empty"
```
Comment (erdemedeiros) [inline Jenkinsfile:47]: Can we have capital for the fist letter of the commit message?

Comment (salaboy) [reply Jenkinsfile:47]: @erdemedeiros good point.. not sure if that is because some kind of convention that github understands..

Comment (salaboy) [reply Jenkinsfile:47]: @erdemedeiros we got that line from how JX is doing releases.. we might want to ask them before changing it

**Pull request merged**